pydoctor/test/test_astbuilder.py

```diff
@@ -449,6 +458,38 @@ def f(): pass
     assert mod_b.resolveName('f') == mod_a.contents['f']
 
 
+@systemcls_param
+def test_import_from_package(systemcls: Type[model.System]) -> None:
+    """Importing from a package should look in __init__ module."""
+    system = systemcls()
+    system.ensurePackage('a')
+    mod_a = fromText('''
+    def f(): pass
+    ''', modname='__init__', parent_name='a', system=system)
+    mod_b = fromText('''
+    from a import f
+    ''', modname='b', system=system)
+    assert mod_b.resolveName('f') == mod_a.contents['f']
+
+
+@systemcls_param
+def test_import_module_from_package(systemcls: Type[model.System]) -> None:
+    """Importing a module from a package should not look in __init__ module."""
```

Comment:
I guess that this is just a  regression tests ?

```suggestion
    """
    Importing a module from a package should not look in __init__ module.
    
    In this test the following hierarchy is constructed:
    a/__init__.py
    a/b.py
    c.py
    
    When c.py has content as below, `f` is resolved as a member if `b.py`:
        from a import b
        f = b.f
    """
```

Or maybe change the name of `f` to a name that describe its purpose in the test.

Reply:
> I guess that this is just a regression tests ?

In the sense that this is the test case that demonstrates the regression that was introduced halfway the PR, yes. But it also tests a fundamental property of the `from ... import` handling, so in that sense it's more of a test that was missing instead of one that specifically exists to check for a regression.

I'll expand the documentation.